### ISSUE TITLE
`log` shouldn't return the string that was logged

### ISSUE THREAD

**Issue by cdepillabout** [slamdata/purescript-aff#41]
Why does [`log`](https://github.com/slamdata/purescript-aff/blob/master/src/Control/Monad/Aff/Console.purs#L11) return the string that was logged?

```purescript
-- | Logs any string to the console. This basically saves you
-- | from writing `liftEff $ log x` everywhere.
log :: forall e. String -> Aff (console :: C.CONSOLE | e) String
log s = do
  liftEff $ C.log s
  return s
```

The [`log`](https://github.com/purescript/purescript-console/blob/master/src/Control/Monad/Eff/Console.purs#L12) function from `Control.Monad.Eff.Console` that's being wrapped around returns `Unit`.

```purescript
-- | Write a message to the console.
foreign import log :: forall eff. String -> Eff (console :: CONSOLE | eff) Unit
```

All of the places I'm using `log`, I don't want it to return a string.  I realize I could write `void $ log "error message"`, but I think most uses of `log` won't need the string returned.  I think the type should be changed to this:

```purescript
log :: forall e. String -> Aff (console :: C.CONSOLE | e) Unit
```

Are other people using `log` such that they want it to return the string that has been logged?

**Comment by jdegoes** [comment]
Good point. I think this is a holdover (perhaps?) from when `log` could debug any `a`, not just a string, so you could use it like, `x <- log $ computeX`, where it makes sense. But given that it can only log strings, that doesn't really hold anymore.

**Comment by cdepillabout** [comment]
@jdegoes That makes sense.  I guess it was like Haskell's `Debug.Trace`.

**Comment by jdegoes** [comment]
PRs welcome. :)

**Comment by texastoland** [comment]
I'll try to tackle it. API needs updated to mirror Prelude too.

**Comment by garyb** [comment]
Can we re-export the CONSOLE effect from the module too, if you're making that change? :please:

**Comment by texastoland** [comment]
Fixed in ae9c72f.
